src/core/exporter.js

```diff
@@ -0,0 +1,104 @@
+// module: exporter
+// used in ui/save-html
```

Comment:
We should probably remove this comment, as things used solely by ui/save-html should like in that module... more below.